bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/DefaultSystemChannelTypeProvider.java

```diff
@@ -48,76 +50,224 @@
+            .withStateDescription(
+                    StateDescriptionFragmentBuilder.create().withReadOnly(true).build().toStateDescription())
+            .build();
+
+    /**
+     * Brightness: default system wide {@link ChannelType} which allows changing the brightness from 0-100%
+     */
+    public static final ChannelType SYSTEM_BRIGHTNESS = new ChannelTypeBuilder(BINDING_ID, "brightness", "Brightness")
+            .withItemType("Dimmer")
+            .withStateDescription(
+                    new StateDescription(BigDecimal.ZERO, new BigDecimal(100), null, "%d %%", false, null))
+            .withCategory("Light").build();
+
+    /**
+     * Color: default system wide {@link ChannelType} which allows changing the color
+     */
+    public static final ChannelType SYSTEM_COLOR = new ChannelTypeBuilder(BINDING_ID, "color", "Color")
+            .withItemType("Color").withCategory("ColorLight").build();
+
+    /**
+     * Color-temperature: default system wide {@link ChannelType} which allows changing the color temperature
+     */
+    public static final ChannelType SYSTEM_COLOR_TEMPERATURE = new ChannelTypeBuilder(BINDING_ID, "color-temperature",
+            "Color Temperature")
+                    .withItemType("Dimmer")
+                    .withStateDescription(
+                            new StateDescription(BigDecimal.ZERO, new BigDecimal(100), null, "%d", false, null))
+                    .build();
+
+    // media channels
+
+    /**
+     * Volume: default system wide {@link ChannelType} which allows changing the audio volume from 0-100%
+     */
+    public static final ChannelType SYSTEM_VOLUME = new ChannelTypeBuilder(BINDING_ID, "volume", "Volume")
+            .withItemType("Dimmer").withDescription("Change the sound volume of a device").withStateDescription(
+                    new StateDescription(BigDecimal.ZERO, new BigDecimal(100), null, "%d %%", false, null))
+            .build();
+
+    /**
+     * Mute: default system wide {@link ChannelType} which allows muting and un-muting audio
+     */
+    public static final ChannelType SYSTEM_MUTE = new ChannelTypeBuilder(BINDING_ID, "mute", "Mute")
+            .withItemType("Switch").withDescription("Mute audio of the device").withCategory("SoundVolume").build();
+
+    /**
+     * Media-control: system wide {@link ChannelType} which controls a media player
+     */
+    public static final ChannelType SYSTEM_MEDIA_CONTROL = new ChannelTypeBuilder(BINDING_ID, "media-control",
+            "Media Control").withItemType("Player").withCategory("MediaControl").build();
+
+    /**
+     * Media-title: default system wide {@link ChannelType} which displays the title of a (played) song
+     */
+    public static final ChannelType SYSTEM_MEDIA_TITLE = new ChannelTypeBuilder(BINDING_ID, "media-title",
+            "Media Title")
+                    .withItemType("String").withDescription("Title of a (played) song")
```

Comment:
We agreed on the description in https://github.com/eclipse/smarthome/issues/3756#issuecomment-388822117. But now I would tend to "Title of a (played) media file". It can be the title of a video as well.